js/src/models/model.project.js

```diff
@@ -348,6 +348,15 @@ Wu.Model.Project = Wu.Model.extend({
 			title : "Could not update project", 
 			description : result.error
 		});
+
+		if(result.updated[0] == "slug") return;
+
+		// store on server
+		this.store.name = result.project.name;
+
+		// update slug name
+		!window.testMode && this.setSlug(result.project.name);
```

Comment:
@knutole Yup I got that and refactored code and separated Project Update and Slug Update